### PR TITLE
Make claims add school functionality use wizard pattern

### PR DESCRIPTION
**Pull request by JamieCleare2525**
## Context

- Refactor for claims service to use an AddSchoolWizard

## Changes proposed in this pull request

- Make a Claims version of the AddSchoolWizard (and related steps)
- Refactor code so claims now uses an AddSchoolWizard
- Remove unnecessary SchoolOnboardingForm

## Guidance to review

Sign in as Colin (Support User)
Run through the functionality of adding a new school
Check nothing is broken.

## Link to Trello card

https://trello.com/c/KXQO7TAY/909-refactor-the-add-organisation-form-to-use-the-wizard-pattern

### CODE REVIEW

File: app/controllers/claims/support/schools/add_school_controller.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+class Claims::Support::Schools::AddSchoolController < Claims::Support::ApplicationController
+  include WizardController
+
+  before_action :authorize_school
+  before_action :set_wizard
+
+  def update
+    if !@wizard.save_step
```
Comment (Kizr) [inline app/controllers/claims/support/schools/add_school_controller.rb:8]:
```suggestion
    unless @wizard.save_step
```

Does this improve readability?

File: app/wizards/claims/add_school_wizard.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+module Claims
+  class AddSchoolWizard < BaseWizard
+    def define_steps
+      add_step(SchoolStep)
+      add_step(SchoolOptionsStep) if steps[:school].school.blank?
```
Comment (Kizr) [inline app/wizards/claims/add_school_wizard.rb:5]: Could this cause an undefined method error? Would `fetch` be more appropriate here?

File: config/locales/en/claims/support/schools/add_school.yml
Comment (Kizr) [inline config/locales/en/claims/support/schools/add_school.yml:1]: Please can we add the missing new line?

File: config/locales/en/wizards/claims/add_school_wizard.yml
Comment (Kizr) [inline config/locales/en/wizards/claims/add_school_wizard.yml:1]: Please can we add the missing new line?

**Review by Kizr — COMMENTED** [review]
Running through the review app looks great 🎉 Just a few small questions!

File: app/controllers/claims/support/schools/add_school_controller.rb (hunk 1 above)
Comment (JamieCleare2525) [reply app/controllers/claims/support/schools/add_school_controller.rb:8]: This is in every wizard. I would be reluctant to change this, as it seems to be the accepted format.

File: app/wizards/claims/add_school_wizard.rb (hunk 2 above)
Comment (JamieCleare2525) [reply app/wizards/claims/add_school_wizard.rb]: I've also addressed this in all other wizards

**Review by Kizr — APPROVED** [review]

**Pull request merged**